deployments/operator/rbac/role.yaml

```diff
@@ -5,6 +5,13 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
+- apiGroups:
+  - ""
+  resources:
+  - nodes/proxy
+  verbs:
+  - get
+  - list
```

Comment:
I think this part is wrong and should not be needed at all. For fractional resources, `inteldeviceplugins-gpu-manager-role` is used.

Reply:
I tried it without initially, but the API server prevented operator spawning roles which the operator itself didn't have access to.

Reply:
Looking at this again, I'll double check this. The roles are included in the reconciler.go and I can't recall anymore if this is really needed.

Reply:
I already made and pushed the change to remove the snippet, as it seemed to work without it. But I then remembered to also test with resourcemanagement, there I got this error in the operator:
```
E0330 08:39:26.403430       1 reconciler.go:419] "intel-device-plugins-manager: unable to create ClusterRoleBinding" err=<
	clusterrolebindings.rbac.authorization.k8s.io "gpu-manager-rolebinding" is forbidden: user "system:serviceaccount:inteldeviceplugins-system:default" (groups=["system:serviceaccounts" "system:serviceaccounts:inteldeviceplugins-system" "system:authenticated"]) is attempting to grant RBAC permissions not currently held:
	{APIGroups:[""], Resources:["nodes/proxy"], Verbs:["get" "list"]}
```
So the snippet is required.

Reply:
> So the snippet is required.

It means the operator serviceaccount *always* needs the same RBAC permissions required by resourcemanager and currently cluster admins have no way to drop them. This probably requires further thinking in the future.

Reply:
Yeah, unfortunately. I suppose we can't reconfigure operator's roles when someone creates a gpu crd with resource manager enabled.

Reply:
the alternative I see is that the elevated serviceaccount for gpu resource manager is created during operator deploy time